FirebaseDatabaseSwift/Sources/Codable/DatabaseReference+WriteEncodable.swift

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import FirebaseDatabase
+
+extension Database.Encoder {
+  public static var defaultEncoder: () -> Database.Encoder = {
+    .init()
+  }
+}
```

Comment:
Looked at it further and I'm not convinced wee need the `defaultEncoder` - was there a specific reason you used this vs using `Database.Encoder()` as the default value below?

Reply:
I just thought it would be a nice touch for the consumer of the API to define a singleton preferred configuration. But it's no big deal - I am removing it for the next iteration. :-)